protocol/daemons/types/health_checkable.go

```diff
@@ -119,12 +128,15 @@ func (h *timeBoundedHealthCheckable) HealthCheck() error {
 
 	// If the last successful update was more than 5 minutes ago, report the specific error.
```

Comment:
```suggestion
	// If the last successful update was more than 5 minutes ago, log the specific error.
	// <add comments why we are just logging instead of returning error>
```